### PR TITLE
chore: update README to reflect LTS status

### PR DESCRIPTION
**Pull request by dhmlau**
Update the status of different versions of LB in README. 

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Comment by dhmlau** [comment]
Let me see how to reduce the size of the logo, possibly need to have some html snippets. 

Regardless of the size, the other option is just to show the logo without the `LoopBack` word, because it's a LB repo, so people probably know it's LoopBack. :) 
loopback-sm (1)


Thoughts, @strongloop/loopback-maintainers ?

**Comment by jannyHou** [comment]
> Regardless of the size, the other option is just to show the logo without the LoopBack word, because it's a LB repo, so people probably know it's LoopBack. :)

Personal opinion 🙈: I think it would be better to keep the word right besides the logo, it helps people remember the project name in a visual way.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # loopback-next
 
+![LoopBack4 logo](https://raw.githubusercontent.com/strongloop/loopback.io/gh-pages/images/branding/logo/blue/loopback-sm.png)
+
```
Comment (raymondfeng) [inline README.md]: The logo is too big.

Comment (raymondfeng) [reply README.md]: Maybe the logo should come after other badges?

Comment (dhmlau) [reply README.md]: I can make the change.. 
I was getting inspirations from the github repos of Node and Express, which put their logo at the very top  :) 
https://github.com/expressjs/express
https://github.com/nodejs/node

Comment (raymondfeng) [reply README.md]: Node does not have badges.

Comment (raymondfeng) [reply README.md]: We can remove `loopback-next` title to match `Express` style.

**Pull request merged**